web/src/app/shared/components/pagination/pagination.component.ts

```diff
@@ -75,4 +80,9 @@ export class PaginationComponent implements OnInit, OnChanges {
     this.pageSelected = page;
     this.changePage.emit(page);
   }
+
+  filter(v) {
+    this.filterValue = v.toString();
```

Comment:
Suggestion to improve readability: The variable filterValue is only used in this method. You could remove the global variable and and set a local one in this method: const filterValue = v.toString();